core/domain/entities/routing/data_nodes/core/SitePermissions.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace EventEspresso\core\domain\entities\routing\data_nodes\core;
+
+use EventEspresso\core\services\json\JsonDataNode;
+use EventEspresso\core\services\json\JsonDataNodeValidator;
+
+class SitePermissions extends JsonDataNode
+{
+    const NODE_NAME = 'sitePermissions';
+
+    /**
+     * @param JsonDataNodeValidator $validator
+     */
+    public function __construct(JsonDataNodeValidator $validator)
+    {
+        parent::__construct($validator);
+        $this->setNodeName(SitePermissions::NODE_NAME);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function initialize()
+    {
+        $permissions = [];
+        $site_permissions = apply_filters(
+            'FHEE__EventEspresso_core_domain_entities_routing_data_nodes_core_SitePermissions__initialize__site_permissions',
+            [
+                'use_bulk_edit' => true,
```

Comment:
toggle this to `false` and then refresh the EDTR to see your bulk edit capabilities disappear...

OR...

remove this altogether and then use a WP Roles and Permissions addon to give yourself a new `use_bulk_edit` cap and see this feature return :)

Reply:
🚀